extensions/amp-story/amp-story.md

```diff
@@ -254,53 +277,224 @@ The `bookend-config-src` value is a URL endpoint that returns data for the end s
+  <noscript>
+    <img alt="Bookend heading component" src="img/amp-story-bookend-component-heading.png" />
+  </noscript>
+</amp-img>
+
+
+##### small
+
+The `small` component can be used to link to related articles. This component requires the following fields: `title`, `url`, and optionally an `image`.
+
+```json
+{
+  "type": "small",
+  "title": "This is India an the best places you should go",
+  "url": "http://example.com/article.html",
+  "image": "http://placehold.it/256x128"
+}
+```
+
+<amp-img alt="Bookend small component" layout="fixed"
+src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/amp-story-bookend-component-small.png" width="379" height="192">
+  <noscript>
+    <img alt="Bookend small component" src="img/amp-story-bookend-component-small.png" />
+  </noscript>
+</amp-img>
+
+##### landscape
+
+The `landscape` component can be used for alternative formats of content, like videos. This component requires the following fields: `title`, `category`, `url`, and an `image`.
+
+```json
+{
+  "type": "landscape",
+  "title": "TRAPPIST-1 Planets May Still Be Wet Enough for Life",
+  "url": "http://example.com/article.html",
+  "category": "astronomy",
+  "image": "http://placehold.it/256x128"
+}
+```
+
+<amp-img alt="Bookend landscape component" layout="fixed"
+src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/amp-story-bookend-component-landscape.png" width="388" height="410">
+  <noscript>
+    <img alt="Bookend landscape component" src="img/amp-story-bookend-component-landscape.png" />
+  </noscript>
+</amp-img>
+
+##### portrait
 
-The end screen displays related articles in sections. The heading for the section is obtained from the provided string key name (e.g., "More to Read"); the array of articles for that key are displayed for that section. The domain and favicon of each linked article are automatically parsed and fetched from the specified URL for each piece of content.
+The `portrait` component can be used to link to other stories. This component requires the following fields: `category`, `url`, and an `image`.
 
-These are configured in the `related-articles` field of the response object.
+```json
+{
+  "type": "portrait",
+  "category": "Science",
+  "url": "http://example.com/article.html",
+  "image": "http://placehold.it/312x416"
+}
+```
+
+<amp-img alt="Bookend portrait component" layout="fixed"
+src="https://github.com/ampproject/amphtml/raw/master/extensions/amp-story/img/amp-story-bookend-component-portrait.png" width="382" height="522">
+  <noscript>
+    <img alt="Bookend portrait component" src="img/amp-story-bookend-component-portrait.png" />
+  </noscript>
+</amp-img>
+
+##### cta-link
+
+The `cta-link` component lets you specify links for call to actions (e.g., `Read More` or `Subscribe`). This component has a `links` key, which specifies an array of links. Each link is an object with a `text` and `url` values.
```

Comment:
Can you style these bookend buttons? For example, if I only have one button, it's not center-aligned, which is what I want.

Reply:
We don't let publishers style any components of the bookend, but we did think of that request you mention. It will be implemented eventually and can be followed here #15391.